src/module_utils.hpp

```diff
@@ -29,6 +29,12 @@ inline std::vector<std::string> split(std::string const& input)
     return values;
 }
 
+// checks if a string starts with a given substring
+inline bool startswith(std::string const& s, std::string const& ss)
```

Comment:
the variable names could be more descriptive to help document usage. I _guess_ `ss` is the substring? but not sure w/o looking at code